runtime/runtime.c

```diff
@@ -157,6 +157,10 @@ INLINE float f32_max(float a, float b) {
     return a > b ? a : b;
 }
 
+INLINE float f32_floor(float a) {
+    return floor(a);
```

Comment:
is this doing an implicit conversion to a double?

Reply:
I think so, yeah! We could definitely use `floorf()` instead.